presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedAccessControl.java

```diff
@@ -320,11 +348,19 @@ public void checkCanDropView(ConnectorSecurityContext context, SchemaTableName v
     @Override
     public void checkCanCreateViewWithSelectFromColumns(ConnectorSecurityContext context, SchemaTableName tableName, Set<String> columnNames)
     {
-        // TODO: implement column level permissions
-        if (!checkTablePermission(context, tableName, SELECT)) {
+        if (INFORMATION_SCHEMA_NAME.equals(tableName.getSchemaName())) {
```

Comment:
It would nice to have a test for that each user has `SELECT` to `INFORMATION_SCHEMA` with grant option.